storm-core/src/jvm/backtype/storm/utils/StormBoundedExponentialBackoffRetry.java

```diff
@@ -44,7 +44,7 @@ public StormBoundedExponentialBackoffRetry(int baseSleepTimeMs, int maxSleepTime
         expRetriesThreshold = 1;
         while ((1 << (expRetriesThreshold + 1)) < ((maxSleepTimeMs - baseSleepTimeMs) / 2))
             expRetriesThreshold++;
-        LOG.info("The baseSleepTimeMs [" + baseSleepTimeMs + "] the maxSleepTimeMs [" + maxSleepTimeMs + "] " +
+        LOG.debug("The baseSleepTimeMs [" + baseSleepTimeMs + "] the maxSleepTimeMs [" + maxSleepTimeMs + "] " +
```

Comment:
It would be nice to use the parameterized call, so that Strings are not constructed unnecessarily:

```Java
Log.debug("The baseSleepTimeMs [{}] ... ", baseSleepTimeMs, ...);
```